sa/db-users/boulder_sa.sql

```diff
@@ -55,7 +55,7 @@ GRANT SELECT ON incidents TO 'sa_ro'@'localhost';
 GRANT SELECT ON certificateStatus TO 'ocsp_resp'@'localhost';
 
 -- Revoker Tool
-GRANT SELECT ON registrations TO 'revoker'@'localhost';
+GRANT SELECT,UPDATE ON registrations TO 'revoker'@'localhost';
```

Comment:
SRE will need to deploy a grant change in staging/prod for this. A sidequest would be renaming the revoker user to something else like boulder_admin.